phpstan-baseline.neon

```diff
@@ -18374,3 +18374,7 @@ parameters:
 			count: 2
 			path: modules/social_features/social_user_export/src/Plugin/Action/ExportUser.php
 
+		-
+			message: "#^Parameter \\#2 \\$node of static method GraphQL\\\\Executor\\\\Values\\:\\:getDirectiveValues\\(\\) expects GraphQL\\\\Language\\\\AST\\\\EnumValueDefinitionNode\\|GraphQL\\\\Language\\\\AST\\\\FieldDefinitionNode\\|GraphQL\\\\Language\\\\AST\\\\FieldNode\\|GraphQL\\\\Language\\\\AST\\\\FragmentSpreadNode\\|GraphQL\\\\Language\\\\AST\\\\InlineFragmentNode, GraphQL\\\\Language\\\\AST\\\\TypeDefinitionNode given\\.$#"
+			count: 1
+			path: modules/custom/social_graphql/modules/social_graphql_oauth2/src/Entity/Server.php
```

Comment:
We should no longer be adding things to the baseline. In PRs the baseline should only get smaller.

We can use an `assert` in the code instead if we expect tests to fail when the condition no longer holds.

Reply:
Asserting won't work here, the second parameter on 'Values::getDirectiveValues()' is missing the 'TypeDefinitionNode' in the docblock comment, which is triggering the PHPStan error. We can remove this from the baseline when webonyx/graphql-php introduces a new release, for more info see: https://github.com/webonyx/graphql-php/pull/1155

Reply:
Then rather than putting it in the baseline lets move the error into `phpstan.neon` itself as ignored error with a link to the issue and PR, that:

- Ensures the issue is automatically muted in all places where we occur (since we know it's an invalid error and not a single unfixed instance)
- Makes it clearer that it's safe to remove this error if PHPStan complains it's no longer present :)